service/src/main/resources/schema.json

```diff
@@ -0,0 +1,278 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Catalog Schema V1",
+  "type": "object",
+  "required": [
+    "dct:title",
+    "dct:description",
+    "dct:creator",
+    "dct:issued",
+    "TerraDCAT_ap:hasDataCollection",
+    "dcat:accessURL",
+    "prov:wasGeneratedBy",
+    "storage",
+    "counts",
+    "contributors"
+  ],
+  "properties": {
+    "TerraCore:id": {
+      "type": "string",
+      "description": "A generated identifier provided by the data catalog"
+    },
+    "dct:title": {
+      "type": "string",
+      "description": "A user provided title for the dataset"
+    },
+    "dct:description": {
+      "type": "string",
+      "description": "A user provided description for the dataset"
+    },
+    "dct:creator": {
+      "type": "string",
+    },
+    "dct:issued": {
+      "type": "date-time",
+      "description": "The date and time the dataset was created."
+    },
+    "dct:modified": {
+      "type": "date-time",
+      "description": "The date and time the dataset was last modified"
+    },
+    "dcat:accessURL": {
+      "type": "string",
+      "description": "A URL to the dataset in the underlying storage system"
+    },
+    "TerraDCAT_ap:hasDataUsePermission": {
+      "description": "Full List taken from https://www.ebi.ac.uk/ols/ontologies/duo/terms?iri=http://purl.obolibrary.org/obo/DUO_0000001",
+      "enum": [
+        "TerraCore:NoRestriction",
+        "TerraCore:GeneralResearchUse",
+        "TerraCore:NPOA",
```

Comment:
Are these fields correct? I thought we found that they aren't the correct values to match on for this property. @tcjiang9